pyenvisalink/honeywell_envisalinkdefs.py

```diff
@@ -1,737 +1,764 @@
-    '^02' : {'type' : 'envisalink', 'name': 'Dump Zone Timers', 'description' : 'This command contains the raw zone timers used inside the Envisalink. The dump is a 256 character packed HEX string representing 64 UINT16 (little endian) zone timers. Zone timers count down from 0xFFFF (zone is open) to 0x0000 (zone is closed too long ago to remember). Each ''tick'' of the zone time is actually 5 seconds so a zone timer of 0xFFFE means ''5 seconds ago''. Remember, the zone timers are LITTLE ENDIAN so the above example would be transmitted as FEFF.','handler' : 'command_response'},
-    '^03' : {'type' : 'envisalink', 'name': 'Keypress to Specific Partition', 'description' : 'This will send a keystroke to the panel from an arbitrary partition. Use this if you don''t want to change the TPI default partition.' ,'handler' : 'command_response'},
-    '^0C' : {'type' : 'envisalink', 'name': 'Response for Invalid Command', 'description' : 'This response is returned when an invalid command number is passed to Envisalink', 'handler': 'command_response'}
+    "Login:": {"name": "Login Prompt", "description": "Sent During Session Login Only.", "handler": "login"},
+    "OK": {"name": "Login Success", "description": "Send During Session Login Only, successful login", "handler": "login_success"},
+    "FAILED": {"name": "Login Failure", "description": "Sent During Session Login Only, password not accepted", "handler": "login_failure"},
+    "Timed Out!": {
+        "name": "Login Interaction Timed Out",
+        "description": "Sent during Session Login Only, socket connection is then closed",
+        "handler": "login_timeout",
+    },
+    "%00": {"name": "Virtual Keypad Update", "description": "The panel wants to update the state of the keypad", "handler": "keypad_update"},
+    "%01": {"type": "zone", "name": "Zone State Change", "description": "A zone change-of-state has occurred", "handler": "zone_state_change"},
+    "%02": {
+        "type": "partition",
+        "name": "Partition State Change",
+        "description": "A partition change-of-state has occured",
+        "handler": "partition_state_change",
+    },
+    "%03": {
+        "type": "system",
+        "name": "Realtime CID Event",
+        "description": "A system event has happened that is signaled to either the Envisalerts servers or the central monitoring station",
+        "handler": "realtime_cid_event",
+    },
+    "%FF": {
+        "name": "Envisalink Zone Timer Dump",
+        "description": "This command contains the raw zone timers used inside the Envisalink. The dump is a 256 character packed HEX string representing 64 UINT16 (little endian) zone timers. Zone timers count down from 0xFFFF (zone is open) to 0x0000 (zone is closed too long ago to remember). Each "
+        "tick"
```

Comment:
Why was the string broken up this way with single or only a couple words on each line?

Reply:
One of the code formatters picked it up for line length.

Reply:
Yes, I know it's breaking up long lines but it seems to have done the wrong thing in cases where there were apostrophes in the string.  E.g. a string of:

```
"This is a 'test' string."
```

is being broken up as:

```
"This is a "
"test"
"string."
```

Reply:
Kk.

I was thinking of restarting the PR as it's all messy due to me never signing the commits.  Also too much has creeped into one PR.

Reply:
So I may just open a bunch of seperate PRs.  I'll look at - this likely Monday night.

Reply:
Sounds good.  I might have a go at getting some of these tools setup as pre-commit hooks over the weekend in which case you may find that some of the linting stuff is already updated.

Reply:
Sounds good.  I'll open a pr with fixes and improvements but no linting.